instrumentation/apache-camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/CamelRoutePolicy.java

```diff
@@ -86,4 +90,16 @@ public void onExchangeDone(Route route, Exchange exchange) {
       LOG.warn("Failed to capture tracing data", t);
     }
   }
+
```

Comment:
Please add override annotation:

```suggestion

  @Override
```